Tajpuriya grammer - Sheet1.csv

```diff
@@ -0,0 +1,41 @@
+English,Tajpuriya/Rajbanshi,English,Tajpuriya/Rajbanshi

```

Comment:
Can you please remove the duplicate header - there should only be one for English and one for Tajpuriya

Reply:
When you do this make sure that the extra commas on lines 2-end are removed